README.md

```diff
@@ -67,6 +69,13 @@ Three different type of scripts are included depending of what you want to downl
 `Unique` : Made to download individual videos made by creators (naming scheme: `Title - Uploader - 20201231/Title - Uploader - 20191231 [VIDEOID].ext`).
 
 
+## Region-restricted videos?
+
+`--ignore-errors` and `--download-archive archive.log` will silently skip such videos.
+
+Using a VPN, switch between US and Germany. Or in general: if you follow a specific country's content, switch to that country (e.g. switch to France if you follow French content)
```

Comment:
Replace with:

Using a VPN, switch between US and Germany, and if you follow a specific country's content, also switch to that country (e.g. switch to France if you follow French content).